contrib/completion/bash/docker

```diff
@@ -70,9 +85,67 @@ __docker_container_ids() {
 	COMPREPLY=( $(compgen -W "${containers[*]}" -- "$cur") )
 }
 
+__docker_image_generate_completions() {
+	local counter=$( __docker_pos_first_nonflag $( __docker_to_alternatives "$options_with_args" ) )
+
+	if [ $cword -eq $counter ]; then
+
+		echo "$cur" | grep -q ':$'
+		if [ $? -eq 0 ]; then
+			__docker_image_repos_and_tags
+		else
+			case "${DOCKER_COMPLETION_SHOW_IMAGE_IDS:-"none"}" in
+				all)
+					case "${DOCKER_COMPLETION_SHOW_TAGS:-"no"}" in
+						yes)
+							__docker_image_repos_and_tags_ids_with_intermediates
```

Comment:
This function exists under a different name.
Corresponding execution path errors out with

    __docker_image_repos_and_tags_ids_with_intermediates: command not found